Entities/MOSRotating.cpp

```diff
@@ -1369,7 +1369,7 @@ bool MOSRotating::DeepCheck(bool makeMOPs, int skipMOP, int maxMOPs)
                 {
                     tally -= 1.0;
                     (*itr)->SetPos((*itr)->GetPos() - m_Vel.GetNormalized() * depth);
-                    (*itr)->SetVel(Vector(velMag * splashDir * PosRand(), -velMag * PosRand()));
+					(*itr)->SetVel(Vector(velMag * splashDir * PosRand(), -velMag * PosRand()));
```

Comment:
Didn't wanna use a min, max random here, or just didn't notice it?

Reply:
this is one of the cases that has potential for min>max.
generally, putting unknown variables into the new random functions should only be done after ensuring min < max.

Reply:
Ah yeah, cause vel can be negative. That's fair.